source/frontend/src/features/documents/documentUpload/DocumentUploadForm.tsx

```diff
@@ -107,6 +103,7 @@ const DocumentUploadForm: React.FunctionComponent<
                 field="documentTypeId"
                 options={documentTypes}
                 onChange={props.onDocumentTypeChange}
+                disabled={documentTypes.length === 1}
```

Comment:
what is this for?

Reply:
Makes the combo box a field if there is only one option. Better than clicking the combobox and not be able to change anything